### ISSUE TITLE
I think there is a small bug in decompression

### ISSUE THREAD

**Issue by danil179** [vhanla/lzrw1#1]
In this line:
https://github.com/vhanla/lzrw1/blob/62e50aea1fcfd0d82723b00ae7ad368fc79e55a9/lzrw1/Program.cs#L329

If I compare with the python implementation:
https://github.com/nmantani/python-lzrw1-kh/blob/0c56e5a3697862587d20b3213780c297354c0d9f/lzrw1_kh/__init__.py#L242

I think it should be ```+buffer[x + 2] + 15``` instead of ```| buffer[x + 2] + 15```. Apart from that I tested it and the decompression works well.

Let me add that this is caused by operator precedence: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/

Basically, it first doing the +15 part and then the bitwise OR. This is a different calculation if it exceeds the value of max of 1 byte.
